src/main/java/org/fundacionjala/at15/katas/pokerhands/mauricio/Game.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.at15.katas.pokerhands.mauricio;
+
+public class Game {
+    private PokerHand h1;
+    private PokerHand h2;
+    private final int tie3 = 3;
+    private final int tie4 = 4;
+
+    public Game(PokerHand hand1, PokerHand hand2) {
+        this.h1 = hand1;
+        this.h2 = hand2;
+    }
+
+    public String whoWins() {
+        String winner = "";
+
+        if (flushCards() == 1) { //Ordered from high to low poker hand rank
+            winner = "Black wins";
+        } else if (flushCards() == 2) {
+            winner = "White wins";
+        } else if (cardsWithTheSameValue() == 1) {
+            winner = "Black wins";
+        } else if (cardsWithTheSameValue() == 2) {
+            winner = "White wins";
+        } else if (highCard() == 1) {
+            winner = "Black wins";
+        } else if (highCard() == 2) {
+            winner = "White wins";
+        }
+        return winner;
+    }
+
+    public int highCard() {
```

Comment:
At some point, `highCard()`, `cardsWithTheSameValue()` and `flushCards()` have almost same implementation. You could improve it by moving repeated code to another function, or even better, leverage inheritance to segregate common code to multiple implementations